pandas/tests/io/test_common.py

```diff
@@ -89,6 +107,68 @@ def test_iterator(self):
         tm.assert_frame_equal(first, expected.iloc[[0]])
         tm.assert_frame_equal(concat(it), expected.iloc[1:])
 
+    @pytest.mark.parametrize('reader, module, path', [
```

Comment:
Yeah, going to do it on top #16292. We'll still need it for pathlib on 3.5, which doesn't implement __fspath__.